### PR TITLE
Insurance Pilot. Added risk indicators for events in MPA. VV-389

### PR DESCRIPTION
**Pull request by rdgfuentes**

### CODE REVIEW

**Review by fertandil87 — APPROVED** [review]

File: apps/vessel-history/features/terminology/terminology-encounter-events.module.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*
```
Comment (nerik) [inline apps/vessel-history/features/terminology/terminology-encounter-events.module.css:1]: This file could probably be removed

**Review by nerik — APPROVED** [review]
Bunch of nitpicks here and there, looks good otherwise!

File: apps/vessel-history/features/profile/Profile.tsx [hunk 2]
```diff
@@ -186,7 +202,7 @@ const Profile: React.FC = (props): React.ReactElement => {
             vessel={vessel}
             lastPosition={lastPosition}
             lastPortVisit={lastPortVisit}
-            onMoveToMap={() => setActiveTab(tabs?.[2])}
+            onMoveToMap={() => setActiveTab(tabs?.[3])}
```
[the remark this answers is not in the document]
Comment (rdgfuentes) [reply apps/vessel-history/features/profile/Profile.tsx:205]: Good catch, I'll rather assign the tabs to constants and use them, it'll be even more clear.

File: apps/vessel-history/features/risk-indicator/risk-indicator.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import { Fragment, useCallback, useMemo, useState } from 'react'
+import { IconButton } from '@globalfishingwatch/ui-components'
+import ActivityEvent from 'features/profile/components/activity/ActivityEvent'
+import { RenderedEvent } from 'features/vessels/activity/vessels-activity.selectors'
+import { Voyage } from 'types/voyage'
+import styles from './risk-indicator.module.css'
+
+/* eslint-disable-next-line */
```
[the remark this answers is not in the document]
Comment (rdgfuentes) [reply apps/vessel-history/features/risk-indicator/risk-indicator.tsx:8]: It was added by the `nx generate @nrwl/next:component`  when I created them, I'll remove it here and in the other components too it seems it's not necessary 👍

**Pull request merged**